src/change-listeners.js

```diff
@@ -198,6 +200,13 @@ class ChangeListeners extends Process {
     }
   }
 
+  async destroy(dbNames) {
+    let listeners = await this._getByDBNames(dbNames)
+    listeners.map(async (listener) => {
+      await this._getAndDestroy(listener._id)
+    })
+  }
+
   _processChange(change, dbName, requests) {
```

Comment:
I'd recommend adding a unit test to _changesForListener in https://github.com/redgeoff/spiegel/blob/master/test/spec/change-listeners.js that stubs _changesArray to generate an not_found error and then check for the DatabaseNotFoundError. (This project uses sinon for mocking)